lib/main.dart

```diff
@@ -35,65 +39,253 @@ class _MyHomePageState extends State<MyHomePage> {
   Widget build(BuildContext context) {
     return Scaffold(
           backgroundColor: Theme.of(context).colorScheme.onPrimaryContainer,
-          body: Column(
-            crossAxisAlignment: CrossAxisAlignment.start,
-            children: [
-              Stack(
-                children: [
-                  const Image(
-                    image: AssetImage('assets/MarioBros_backdrop.jpg'),
-                    colorBlendMode: BlendMode.srcATop,
-                  ),
-                  SafeArea(
-                    child: IconButton(
-                      icon: const Icon(Icons.arrow_back_ios_new_rounded),
-                      onPressed: (){
+          body: SingleChildScrollView(
+            child: Column(
+              crossAxisAlignment: CrossAxisAlignment.start,
+              children: [
+                SizedBox(
+                  height: MediaQuery.of(context).size.height * 0.6,
+                  child: Stack(
+                    clipBehavior: Clip.none,
+                    children: [
+                      Positioned(
+                        child: ClipRRect(
+                          borderRadius: BorderRadius.circular(10),
+                          child: const Image(
+                            image: AssetImage('assets/MarioBros_backdrop.jpg'),
+                            colorBlendMode: BlendMode.srcATop,
+                          ),
+                        ),
+                      ),
+                      Positioned(
+                        child: SafeArea(
+                          child: IconButton(
+                            icon: const Icon(Icons.arrow_back_ios_new_rounded),
+                            onPressed: (){
 
-                      },
-                    ),
-                  ),
-                ],
-              ),
-              Row(
-                children: [
-                  Padding(
-                    padding: const EdgeInsets.fromLTRB(12, 0, 0, 0),
-                    child: ClipRRect(
-                      borderRadius: BorderRadius.circular(10),
-                      child: Container(
-                        decoration: BoxDecoration(
-                          border: Border.all(
-                            color: Theme.of(context).colorScheme.onPrimaryContainer,
-                            width: 3.0,
-                          )
+                            },
+                          ),
                         ),
-                        child: const Image(
-                            image: AssetImage('assets/MarioBros_poster.jpg'),
-                            width: 150,
+                      ),
+                      Positioned(
+                        width: MediaQuery.of(context).size.width,
+                        bottom: 0,
+                        child: Row(
+                          children: [
+                            Padding(
+                              padding: const EdgeInsets.fromLTRB(12, 0, 0, 0),
```

Comment:
LRTB defaults value is 0 , so you change this to EdgeInsets.only(left: 12.0) and also add this value to a constant file

Reply:
And also you missed a trailing comma after the last 0